frontend/src/components/maps/leaflet/InfoSlideOut/ThreeColumnItem.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { Col, ListGroup, Row } from 'react-bootstrap';
+import { Label } from 'components/common/Label';
+
+const InnerRow = styled(Row)`
+  margin: 0px;
+  width: 300px;
+`;
+
+const LeftCol = styled(Col)`
+  width: 110px;
+  max-width: 110px;
+  padding-right: 10px;
+  padding-left: 0px;
+`;
+
+const CenterCol = styled(Col)`
+  max-width: 1px;
+  padding: 0px;
+  background-color: rgba(96, 96, 96, 0.2);
+`;
+
+const RightCol = styled(Col)`
+  padding-left: 10px;
+  padding-right: 0px;
+`;
+
+interface IThreeColItem {
+  leftSideLabel: string;
+  rightSideItem: any;
```

Comment:
not sure if this "any" is an issue, I think I can change this to "string | undefined" if need be

Reply:
I'd always prefer the attempt to be as close to the expected value as possible.